bindings/matlab/iio-wrapper.h

```diff
@@ -1,4 +1,56 @@
 #ifndef MATLAB_LOADLIBRARY
 #define MATLAB_LOADLIBRARY
 #include <iio.h>
+
+#ifdef _WIN32
+#   ifdef LIBIIO_EXPORTS
+#	define __api __declspec(dllexport)
+#   else
+#	define __api __declspec(dllimport)
+#   endif
+#elif __GNUC__ >= 4 && !defined(MATLAB_MEX_FILE) && !defined(MATLAB_LOADLIBRARY)
+#   define __api __attribute__((visibility ("default")))
```

Comment:
I made a patch for your patch: https://gist.github.com/pcercuei/e2e4cd23dfa09e4e4aaa9140e05fba1a
With this change it builds fine on Linux and the symbols are available.